modules/gce_worker_group/main.tf

```diff
@@ -18,6 +18,10 @@ variable "gcloud_cleanup_loop_sleep" {
   default = "1s"
 }
 
+variable "gcloud_cleanup_opencensus_sampling_rate" {}
+
+variable "gcloud_cleanup_opencensus_tracing_enabled" {}
```

Comment:
I'd suggest setting this to `"false"` as a default, to be a bit more explicit about the expected default behaviour when nothing is specified.